monitoring/perf_context.cc

```diff
@@ -10,6 +10,296 @@
 
 namespace ROCKSDB_NAMESPACE {
 
+/*
+ * Please add new metrics to this macro and appropriate fields will be copied,
+ * and/or emitted when converted to string.
+ * When people need to add new metrics please add an optional multi-line C style
+ * comment on what the metric semantics are and enclose the name of the specific
+ * metric within defCmd(). The position of the field will be dicated by the
+ * order in which the macros are enumerated and the offsets of the fields will
+ * be matched against ''PerfContextByLevelBase'' declared in perf_context.h.
+ */
+#define DEF_PERF_CONTEXT_LEVEL_METRICS(defCmd)                                          \
+  /* # of times bloom filter has avoided file reads, i.e., negatives. */                \
+  defCmd(bloom_filter_useful) /* # of times bloom FullFilter has not avoided            \
+                                 the reads */                                           \
```

Comment:
I would omit the comments describing the fields since they are already in perf_context.h

Reply:
This is fixed in the latest version,